eZ/Bundle/EzPublishDebugBundle/Collector/SiteAccessCollector.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
```

Comment:
done in https://github.com/ezsystems/ezpublish-kernel/pull/1790/commits/2a24de3f00a0d953265789def08931b1b2f9240e